crates/holochain_types/src/app.rs

```diff
@@ -145,36 +146,20 @@ impl AppBundleSource {
 /// Information needed to specify a Dna as part of an App
 #[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
 pub struct InstallAppDnaPayload {
-    /// The path of the DnaFile
-    pub path: Option<PathBuf>, // This is a deprecated field should register dna contents via register
-    /// The path of the DnaFile
-    pub hash: Option<DnaHash>, // When path is removed, this will become non-opt
+    /// The hash of the DnaFile
```

Comment:
```suggestion
    /// The hash of the DnaBundle
```

Reply:
Actually we decided on:

```suggestion
    /// The hash of the DNA
```